ci/deploy-vmck

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+function clean {
+    vagrant destroy -f || echo "vagrant destroy failed, but we don't care"
+}
+trap clean  EXIT
+
+set -ex
+
+vagrant up
+
+vagrant ssh <<'EOF'
```

Comment:
I don't like this `ssh <<EOF` thing, it's scary to the reader. Let's just do:
```shell
vagrant ssh -- /opt/vmck/ci/test-pytest.sh
vagrant ssh -- /opt/vmck/ci/test-alive-on-cluster.sh
```

And make sure they are executable, so you don't have to write `bash` in this script.